src/WarcraftLegacies.Source/Quests/Dalaran/QuestTheramore.cs

```diff
@@ -45,6 +46,14 @@ public sealed class QuestTheramore : QuestData
     protected override string RewardDescription =>
       "Gain control of all units at Theramore and teleport all of your units within Dalaran to Theramore. Dalaran becomes hostile";
 
+    /// <inheritdoc />
+    protected override string PenaltyFlavour =>
+      "The people of Dalaran have to abandon their city. The colony of Theramore is founded in Kalimdor.";
```

Comment:
This is actually perfectly solid. Can I make it slightly more emotive, like:
```suggestion
      "Dalaran has fallen. Those who managed to survive its destruction travel west, to the distant lands of Kalimdor. They hope that this new world will treat them more kindly than the one they left behind.";
```